packages/core/test/unit/model/find-by-pk.test.js

```diff
@@ -33,5 +34,43 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       Model.findOne.should.not.have.been.called;
       Sequelize.Model.findOne.should.have.been.called;
     });
+
+    it('should use composite primary key when querying table has one', async () => {
+      const Model = current.define('model', {
+        pk1: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+        },
+        pk2: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+        },
+      });
+
+      Model.findOne = sinon.stub();
```

Comment:
```suggestion
      sinon.stub(Model, 'findOne');
```

... otherwise Sinon doesn't know that it should restore `Model.findOne` after the test.

Reply:
Ah, you're copy/pasting from the above code, and `Model` is thrown away after the test. I'd still argue that my suggestion is generally better though 😇